src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movie_session")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToOne
+    private Movie movie;
+
+    @OneToOne
```

Comment:
Only one session can take place in one cinemaHall? (not in one time)

Reply:
Also changed to ManyToOne